react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -100,6 +105,15 @@ void addPlugin(IAztecPlugin plugin) {
         }
     }
 
+    @Override
+    public boolean onTextContextMenuItem(int id) {
+        if (id == android.R.id.paste && shouldHandleOnPaste) {
```

Comment:
Should we intercept the _Paste as plain text_ event here (`android.R.id.pasteAsPlainText`), and call `onPaste` passing the instructions to not send the HTML version of the content up to the event?

Reply:
Yes, good suggestion. We can pass a boolean to onPaste, and send `pastedHtml` as an empty string in that case.